satpy/composites/spectral.py

```
@@ -16,55 +16,164 @@
-    affects true colour RGBs, because vegetation looks brown rather than green.
-    To make vegetation look greener again, this corrector allows
-    to simulate the green band as a fraction of two or more other channels.
+    The green band in FCI and AHI (and other bands centered at 0.51 microns) deliberately
+    misses the chlorophyll spectral reflectance local maximum at 0.55 microns
+    in order to focus on aerosol and ash rather than on vegetation. This
+    affects true colour RGBs, because vegetation looks brown rather than green
+    and barren surface types typically gets a reddish hue.
 
-    To be used, the composite takes two or more input channels and a parameter
-    ``fractions`` that should be a list of floats with the same length as the
-    number of channels.
+    To correct for this the hybrid green approach proposed by Miller et al. (2016, :doi:`10.1175/BAMS-D-15-00154.2`)
+    is used. The basic idea is to include some contribution also from the 0.86 micron
+    channel, which is known for its sensitivity to vegetation. The formula used for this is::
 
-    For example, to simulate an FCI corrected green composite, one could use
-    a combination of 93% from the green band (vis_05) and 7% from the
-    near-infrared 0.8 µm band (vis_08)::
+      hybrid_green = (1 - F) * R(0.51) + F * R(0.86)
 
-      corrected_green:
-        compositor: !!python/name:satpy.composites.ahi.GreenCorrector
-        fractions: [0.93, 0.07]
+    where F is a constant value. The validation against VIIRS presented in the paper suggests
+    F = 0.07 as an optimal value to reconstruct a green band optimized for true color imagery
+    from AHI. This is the default value currently used for the hybrid green correction in Satpy.
+
+    For example, to construct the hybrid green for AHI, one should use
+    a combination of 93% from the green band (B02) and 7% from the
+    near-infrared 0.85 µm band (B04)::
```

I think we should mention in this docstring somewhere that the default in satpy is 15%: the mention of 7% can be misleading.

---

That was indeed misleading, has been clarified now.